src/main/scala/com/dwolla/postgres/init/repositories/UserRepository.scala

```diff
@@ -41,10 +42,10 @@ object UserRepository {
                                     password: Password): Kleisli[F, Session[F], Command[Void]] = Kleisli {
       _
         .prepare(UserQueries.checkUserExists)
-        .use(_.unique(username))
+        .use(_.option(username))
         .flatMap {
-          case 0 => Logger[F].info(s"Creating user $username") >> UserQueries.createUser(username, password).pure[F]
-          case count => Logger[F].info(s"Found and updating $count user named $username") >> UserQueries.updateUser(username, password).pure[F]
+          case Some(_) => Logger[F].info(s"Found and updating user named $username").as(UserQueries.updateUser(username, password))
```

Comment:
Out of curiosity, is `.as()` equivalent to `>>` in this context? I see the difference in implementation is map vs flatmap, but guessing it doesn't matter here because the block is inside a flatmap? (line 45)